grids/AccountsGridView.php

```diff
@@ -157,23 +154,58 @@ public function init()
                                             'disabled' => true,
                                             'data-toggle' => 'tooltip',
                                             'data-placement' => 'right',
-                                            'title' => 'No assets available on this account!',
+                                            'title' => 'Transfer can be done only when investment is accepted for related crowdfunding campaign',
```

Comment:
consider using `Yii::t('XcoinModule.base', '')` for translations

Reply:
Let's resolve translations later on translation branch